<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeightedPCADialog</class>
 <widget class="QDialog" name="WeightedPCADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeightedPCA</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_WeightedPCA">
             <property name="text">
              <string> WeightedPCA : </string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_X">
             <item>
              <widget class="QLabel" name="label_X">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of points taken into account for local surface fitting.&lt;br/&gt;Low value leads to a faster computation time but can decrease quality in case of important sampling anisotropy or noise level.&lt;br/&gt;&lt;br/&gt;Typical: 100&lt;br/&gt;For scattered point clouds : 50&lt;br/&gt;For very noisy point clouds : 300&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="whatsThis">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of points taken into account for local surface fitting.&lt;br/&gt;Low value leads to a faster computation time but can decrease quality in case of important sampling anisotropy or noise level.&lt;br/&gt;&lt;br/&gt;Typical: 100&lt;br/&gt;For scattered point clouds : 50&lt;br/&gt;For very noisy point clouds : 300&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Number of neighbours</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_X"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_Y">
             <item>
              <widget class="QLabel" name="label_Y">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The sensor noise, in meters.&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;For bad sensor: 0.03&lt;br/&gt;For good sensor : 0.005&lt;br/&gt;For artificial point cloud: 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="whatsThis">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The sensor noise, in meters.&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;For bad sensor: 0.03&lt;br/&gt;For good sensor : 0.005&lt;br/&gt;For artificial point cloud: 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Noise (m)</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Y"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_Z">
             <item>
              <widget class="QLabel" name="label_Z">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum curvature radius to define a sharp feature, in meters.&lt;br/&gt;&lt;br/&gt;For curved object: 0.1&lt;br/&gt;For piecewise planar objects: inf&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="whatsThis">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum curvature radius to define a sharp feature, in meters.&lt;br/&gt;&lt;br/&gt;For curved object: 0.1&lt;br/&gt;For piecewise planar objects: inf&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Curvature (m)</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Z"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>For any help, mouse over any parameter's name</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WeightedPCADialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WeightedPCADialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
